docs/how-to/deploy/deploy-spaces.md

```diff
@@ -0,0 +1,53 @@
+# Deploy on Juju spaces
+
+The Charmed Apache Kafka operator supports [Juju spaces](https://documentation.ubuntu.com/juju/latest/reference/space/index.html) to separate network traffic for:
+
+- **Internal communications**, including inter-broker and broker-to-controller communications.
+- **Client**, including traffic between broker and clients (producers and consumers).
+
+## Prerequisites
+
+* Charmed Apache Kafka 4
+* Configured network spaces
+  * See [Juju | How to manage network spaces](https://documentation.ubuntu.com/juju/latest/reference/juju-cli/list-of-juju-cli-commands/add-space/)
```

Comment:
Todo: This link does not lead to a How-to guide, but to a Juju CLI command reference instead. Replace the link or rename it.